workloads/kube-burner/common.sh

```diff
@@ -12,21 +17,34 @@ fi
 export TOLERATIONS="[{key: role, value: workload, effect: NoSchedule}]"
 export UUID=$(uuidgen)
 
-log() {
-  echo -e "\033[1m$(date "+%d-%m-%YT%H:%M:%S") ${@}\033[0m"
-}
```

Comment:
Why this change? the original implementation should work in every platform with a recent version of bash. In addition, where are the bold and normal vars defined?

Reply:
I believe this change was done to keep consistent datetime log across workload, understanding the original implementation was intentional so this will be removed.